packages/react-make-styles/jest.config.js

```diff
@@ -1,5 +1,21 @@
-const { createConfig } = require('@fluentui/scripts/jest/jest-resources');
+// @ts-check
 
-const config = createConfig();
-
-module.exports = config;
+/**
+ * @type {jest.InitialOptions}
+ */
+module.exports = {
+  displayName: 'react-make-styles',
+  preset: '../../jest.preset.js',
+  globals: {
+    'ts-jest': {
+      tsConfig: '<rootDir>/tsconfig.json',
+      diagnostics: false,
+    },
+  },
+  transform: {
+    '^.+\\.tsx?$': 'ts-jest',
+  },
+  coverageDirectory: './coverage',
+  setupFilesAfterEnv: ['./config/tests.js'],
+  snapshotSerializers: ['@fluentui/jest-serializer-make-styles'],
```

Comment:
It looks that there is a last CI failure:

```
ERR!   Module @fluentui/jest-serializer-make-styles in the snapshotSerializers option was not found.
ERR!          <rootDir> is: /mnt/work/1/s/packages/react-make-style
```

It happens because this package does not have `@fluentui/jest-serializer-make-styles` in `dependencies`/`devDependencies`. However it even don't needs it, it is safe to delete this line:

```suggestion
```

---

An issue that tracks it #18706.

Reply:
yup, thx 🙌